.github/workflows/go.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   build:
     strategy:
       matrix:
-        go: ['1.14', '1.15', '1.16', '1.17', '1.18', '1.19']
+        go: ['1.17', '1.18', '1.19', '1.20']
```

Comment:
The x/sys version upgrade in the PR requires go version 1.17.  Removing old versions from the build matrix.